### ISSUE TITLE
google article

### ISSUE THREAD

**Issue by vanslykej** [cookc3174/cookc3174.github.io#1]
In the commit 8bde622c5ce4a04d239308a653189402d3122e70, there are a few issues:
* I would add a little more detail about why specifically students are becoming lazier
* I would either fix the picture or pick a new one since it is blurry
* Possibly more detail about what specifically students look up on google

**Comment by cookc3174** [comment]
Yeah I have been having problems with the picture. I am still in the process of finding a new one.

**Comment by cookc3174** [comment]
I tried to fix the picture, and I also fixed some errors and added a few lines in paragraph 1. I might have to find a new picture because this one is still a little too blurry.

**Comment by vanslykej** [comment]
The changes you made look good, but yeah would just keep working on the picture.

**Comment by cookc3174** [comment]
I added a few things to my writing. I also need to add a few more lines. Could you read it over and give me ideas, like something I might have missed or briefly talked about. I need to make more changes but at the same time it has to make sense.

**Comment by vanslykej** [comment]
I like what you have written, and there is not much that I can recommend, however I would suggest maybe expanding slightly on what people should do instead or how to effectively and strategically use Google to maximize what they can gain from it.
